### PR TITLE
added samples for azure openai sdk and cli

### PR DESCRIPTION
**Pull request by vvatsalya**
# Description
Added samples for azure openai sdk and cli

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/python/foundation-models/azure_openai/data/train.jsonl [hunk 1]
```diff
@@ -0,0 +1,408 @@
+{"prompt": "sentence1\nHe said the foodservice pie business doesn 't fit the company 's long-term growth strategy .\n\nsentence2\n\" The foodservice pie business does not fit our long-term growth strategy .", "completion": "paraphrase<|endoftext|>"}
```
Comment (nagau) [inline sdk/python/foundation-models/azure_openai/data/train.jsonl:1]: Which dataset is this?

File: cli/foundation-models/azure_openai/openai_completions_finetune_pipeline/images/pipeline_completed.png
Comment (nagau) [inline cli/foundation-models/azure_openai/openai_completions_finetune_pipeline/images/pipeline_completed.png:1]: Why this one refers to 0.0.2 version instead?

Comment (vvatsalya) [reply cli/foundation-models/azure_openai/openai_completions_finetune_pipeline/images/pipeline_completed.png:1]: The pipeline component version is `0.0.2`.

File: sdk/python/foundation-models/azure_openai/data/train.jsonl (hunk 1 above)
Comment (vvatsalya) [reply sdk/python/foundation-models/azure_openai/data/train.jsonl:1]: I guess this is some book dataset.

**Pull request closed without merging**